index.rst

```diff
@@ -12,18 +12,633 @@
+
+The conclusion that we can get from this is that most of the filtering is done for diaObjects that have less than 5 diaSources associated. This indicates a transient candidate that is not likely to be astrophysical in origin, although we are dealing with fakes in this situation it is acceptable.
+Another conclusion from this plot is that the number of diaObjects in Pre-convolution is higher, but after filtering it ends up being lower (by a ~300 diaObjects margin) than the other flavors. Our plot also shows that a significant portion of these could be in the bin of 20 or more diaSources, which is interesting.
+
+In the following figure we have a scatter plot of these diaObjects on sky coordinates.
+
+.. figure:: /_static/figures/good_diaObj_zoomed_sky.png
+   :name: good_diaObj_zoomed_sky
+   :target: ../_images/good_diaObj_zoomed_sky.png
+   :alt: Scatter of diaObjects in the sky
+
+   Scatter plot showing the position of DIA source associations for each filter and each DIA flavor or mode. Size of the points is proportional to the number of associated diaSources.
+
+We observe the spatial distribution of the diaObjects and their number of diaSources also displayed as the size of the scatter points. We can observe that the objects are clustered around what could be bright sources in the field. In contrast to the default mode we see that there are less points in the pre-convolution mode, although the difference is subtle.
+
+We can understand that the associated diaSources in this plot should have already a significant cut, and they are mostly equivalent.
+
+
+
+
+Fake source injection analysis
+==============================
+
+Number of matches
+-----------------
+
+We can expand the table that we built before to include the number of fake source matches.
+
+.. _table-nfakematches:
+
+.. table:: Table with number of fake matches per DIA flavor.
+
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    |           | Good diaSrc | Good diaObject | N Fakes Match | contamination |  Matches w/flag cuts|
+    +===========+=============+================+===============+===============+=====================+
+    |   Default |       13244 |           6166 |        4627.0 |       50172.0 |              2391.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    |  Pre-Conv |       10909 |           6381 |        4594.0 |       55649.0 |              1732.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    | Auto Mode |       13727 |           6698 |        4321.0 |       45728.0 |              2348.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+
```

Comment:
Please describe what `contamination` is.

Reply:
Added clarification thanks.